src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetModuleFileName.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Text;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, SetLastError = true, BestFitMapping = false)]
+        private static extern int GetModuleFileName(HandleRef hModule, ref char buffer, int length);
```

Comment:
```suggestion
        private static extern int GetModuleFileName(IntPtr hModule, char* buffer, int length);
```

- There is no point in using HandleRef here
- It is not a good practice to use `ref char` for strings in PInvoke signatures. It obfuscates the intent. (I know there are some PInvokes under libraries that do it, but it does not make it a good practice.)